test/user-profile/helpers.unit.spec.js

```diff
@@ -0,0 +1,85 @@
+import { expect } from 'chai';
+import _ from 'lodash';
+
+import {
+  formTitles,
+  formLinks,
+  handleIncompleteInformation,
+  handleNonSIPEnabledForm,
+  sipEnabledForms,
+} from '../../src/js/user-profile/helpers';
+
+import fullSchema1010ez from '../../src/js/hca/config/form';
+import fullSchema1990 from '../../src/js/edu-benefits/1990-rjsf/config/form';
+import fullSchema1990e from '../../src/js/edu-benefits/1990e/config/form';
+import fullSchema1990n from '../../src/js/edu-benefits/1990n/config/form';
+import fullSchema1995 from '../../src/js/edu-benefits/1995/config/form';
+import fullSchema5490 from '../../src/js/edu-benefits/5490/config/form';
+import fullSchema5495 from '../../src/js/edu-benefits/5495/config/form';
+import fullSchema527EZ from '../../src/js/pensions/config/form';
+import fullSchema530 from '../../src/js/burials/config/form';
+
+import schemas from 'vets-json-schema/dist/schemas';
+
+// Maps schema id to config id
+const schemaToConfigIds = {
+  '10-10EZ': '1010ez',
+  '21P-527EZ': '21P-527EZ',
+  '21P-530': '21P-530',
+  '22-1990': '1990',
```

Comment:
I think the current ids for the education forms are made up, and should match the id in the schema file name.

Reply:
@jbalboni, just to confirm, is HCA the exception in that it should not match? In which case this key will still be required but will be updated with updated education formIds?

Reply:
HCA is the exception in that the form id doesn't match the id in the schema file name (i.e. it isn't uppercase). All the other ids should match.

Reply:
Thanks again, @jbalboni. Sounds good, changed.